Source/WebCore/css/parser/CSSParserToken.h

```diff
@@ -135,7 +135,11 @@ class CSSParserToken {
 
     CSSPropertyID parseAsCSSPropertyID() const;
 
-    void serialize(StringBuilder&, const CSSParserToken* nextToken = nullptr) const;
+    enum class SerializationMode : bool {
+        Normal,
+        CustomProperty // Custom properties don't collapse whitespace and serialize numbers as original strings.
+    };
+    void serialize(StringBuilder&, const CSSParserToken* nextToken = nullptr, SerializationMode = SerializationMode::Normal) const;
```

Comment:
`SerializationMode` could be a constexpr here it seems?

(albeit we don't care that much about performance for serialisation code anyway)